src/nwbinspector/checks/ecephys.py

```diff
@@ -79,3 +79,13 @@ def check_spike_times_not_in_unobserved_interval(units_table: Units, nunits: int
                     "observed intervals."
                 )
             )
+
+
+@register_check(importance=Importance.BEST_PRACTICE_VIOLATION, neurodata_type=Units)
+def check_ascending_spike_times(units_table: Units):
```

Comment:
We are still testing the performance of this, since it's the first data heavy check to be added without options to control how much data is loaded

Reply:
@alessandratrapani Can you add a `nelems` option here to mirror other table tests?

Reply:
ping @alessandratrapani